### PR TITLE
Add pageView to loadDashboardConfiguration

### PR DESCRIPTION
**Pull request by noanu**

### CODE REVIEW

File: src/dashboard.ts [hunk 1]
```diff
@@ -85,6 +85,16 @@ export class Dashboard extends embed.Embed implements IDashboardNode {
      * Validate load configuration.
      */
     validate(config: models.IDashboardLoadConfiguration): models.IError[] {
-      return models.validateDashboardLoad(config);
+      let error = models.validateDashboardLoad(config);
+      return error ? error : this.ValidatePageView(config.pageView);
```
Comment (ranbreuer) [inline src/dashboard.ts]: i think you can do
error || this.validate....

Comment (ArinaHantsis) [inline src/dashboard.ts]: Why isn't ValidatePageView part of validateDashboardLoad?

File: src/dashboard.ts [hunk 2]
```diff
@@ -85,6 +85,16 @@ export class Dashboard extends embed.Embed implements IDashboardNode {
...
+    }
+   
+    /**
+     * Validate that pageView has a legal value: if page view is defined it must have one of the values defined in models.PageView
+     */
+    private ValidatePageView(pageView: models.PageView): models.IError[] {
```
Comment (ArinaHantsis) [inline src/dashboard.ts]: Isn't there any utility class (in TS) existing or new one we can write for verifying whether a value is a valid among all the available values in an emun?

File: test/test.spec.ts [hunk 3]
```diff
@@ -20,6 +20,34 @@ declare global {
...
+
+  const dashboardEmbedConfig = {
+    type: "dashboard",
+    id: "fakeReportId",
+    accessToken: "fakeAccessToken",
+    embedUrl: "fakeEmbedUrl",
+    pageView: pageView
+  };
+
+  powerbi.embed(component[0], <any>dashboardEmbedConfig);
```
Comment (ArinaHantsis) [inline test/test.spec.ts]: Seems redundant as you embed within try{} catch{}

File: src/dashboard.ts (hunk 1 above)
Comment (noanu) [inline src/dashboard.ts]: I think it will return false or true like this - won't it? I want to return the error itself.

Comment (noanu) [inline src/dashboard.ts]: the "error" variable isn't boolean...

Comment (noanu) [inline src/dashboard.ts]: validateDashboardLoad is auto generated in the powerbi-models project...

File: test/test.spec.ts (hunk 3 above)
Comment (noanu) [inline test/test.spec.ts]: The validate is called only on "load" (not on "embed"). In "reality" - the constructor of embed class assigns "onLoad" to the iframe that calls the "load" function when the iframe is ready. That doesn't happen on tests, so we need to call the embed again - when embeding the second time the code flow doesn't wait for the iframe to be ready and calls the load function.

**Pull request merged**